frontend/packages/console-app/src/components/user-preferences/language/getPreferredLanguage.ts

```diff
@@ -0,0 +1,4 @@
+import { PREFERRED_LANGUAGE_LOCAL_STORAGE_KEY } from './const';
+
+export const getPreferredLanguage = (): string =>
```

Comment:
Same here. We should rename this utility so it's clear it's not the current preference. Rather it's the last-known value.

```suggestion
export const getLastLanguage = (): string =>
```